.github/RELEASING.md

```diff
@@ -0,0 +1,66 @@
+# Tauri Releasing Handbook
+
+This handbook contains information about our release pipeline and how to deal with common issues.
+This document is mainly intended for team members responsible for maintaining the project.
+
+- [Covector](#covector)
+- [Version Pull Request](#version-pull-request)
+- [Releasing and Publishing](#releasing-and-publishing)
+- [Publishing failed, what to do?](#publishing-failed-what-to-do)
+
+# Covector
+
+We use [`covector`](https://github.com/jbolda/covector) to manage our version bumps and release pipeline.
+It can be configured in [`.changes/config.json`](../.changes/config.json) which includes how each package should be published step by step.
+
+Some packages can't be published directly using `covector` as it requires to be built on a matrix of platforms
+such as `tauri-cli` prebuilt binraies which is published using using [publish-cli-rs.yml](./workflows/publish-cli-rs.yml)
+and `@tauri-apps/cli` native Node.js modules which is published using using [publish-cli-js.yml](./workflows/publish-cli-js.yml)
+both of which are triggered after `covector` has created a github release for both of them, see `Trigger @tauri-apps/cli publishing workflow`
+and `Trigger tauri-cli publishing workflow` steps in [covector-version-or-publish.yml](./workflows/covector-version-or-publish.yml)
+
+# Version Pull Request
+
+On each pull request merged, [covector-version-or-publish.yml](./workflows/covector-version-or-publish.yml) workflow will run to determine whether to publish or not
```

Comment:
This confused me a lot to be honest, I need to work my way backwards to understand that this is basically saying when you remove all the change files, it will trigger the publish process

Reply:
Sorry about that, I am not very good at writing, how would you rephrase it?

Reply:
I'll try, but I'm not good at it either 😂